docs/projects/tad/reporting-standard/latest.md

```diff
@@ -62,10 +62,21 @@ the examples in the next section.
 A `system_card` contains the following information.
 
 1. `schema_version` (REQUIRED, string). Version of the schema used, for example "0.1a2".
-2. `name` (OPTIONAL, string). Name used to describe the system.
-3. `upl` (OPTIONAL, string). If this algorithm is part of a product offered by the Dutch Government,
+2. `provenance` (OPTIONAL). In case this System Card is generated from another source file, this field
+can capture the historical context of the contents of this System Card.
+
+    1. `modifier` (OPTIONAL, string). Git-hash of the file containing the transformations that where used
+    to fill in this System Card.
+    2. `timestamp` (OPTIONAL, string). A timestamp of the date, time and timezone of generation of this System Card.
+    Timestamp should be given, preferably in UTC (represented as `Z`), in [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html)
+    format, i.e. `2024-04-16T16:48:14Z`.
+    3. `author` (OPTIONAL, string). Name or identifier of the person or system who initiated the
```

Comment:
I'm not sure we should conflate persons with systems. Could we have to separate fields maybe? 
"author" for a person and "repo" (or "tool" or "repo-uri" or just "uri") for a reference to the tool?

Reply:
Added your suggestion.